addons/stock/models/__init__.py

```diff
@@ -20,3 +20,4 @@
 from . import stock_package_level
 from . import stock_package_type
 from . import stock_storage_category
+from . import res_users
```

Comment:
Can you add this to underneath the `res_partner` import so we can at least try to pretend like we're following alphabetical ordering (I see that it's already a bit of a mess :disappointed: )

Reply:
Thanks for pointing it ;)
I have implemented this change